src/main/java/DataBase/DatasourceConfig.java

```diff
@@ -20,8 +20,8 @@ public class DatasourceConfig {
     /*
      * Change database url and login information depending on your database
      * */
-    private static String dbUrl = "jdbc:postgresql://localhost/dbname";
-    private static String user = "username";
+    private static String dbUrl = "jdbc:postgresql://localhost/postgres";
+    private static String user = "postgres";
```

Comment:
hier bitte postgres stehen lassen. ich glaube, zu dieser anpassung steht etwas in der anleitung